### PR TITLE
Release cloud-admin to netlify when tag

### PR DESCRIPTION
**Pull request by himadrisingh**
and release to prod

### CODE REVIEW

File: .github/workflows/cloud-admin.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy Cloud Admin to Netlify
+on:
+  push:
+    tags:
+      - "*"
+    branches: ["netlify"]
```
Comment (himadrisingh) [inline .github/workflows/cloud-admin.yml:6]:
```suggestion
    branches: ["main"]
```

File: .github/workflows/cloud-admin.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy Cloud Admin to Netlify
+on:
+  push:
+    tags:
+      - "*"
+    branches: ["main"]
+    paths:
+      - ".github/workflows/cloud-admin.yml"
+      - "web-admin/**"
+      - "web-local/**"
```
Comment (ericpgreen2) [inline .github/workflows/cloud-admin.yml:10]: Any reason for `web-local` to be included here?

File: .github/workflows/cloud-admin.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```
Comment (ericpgreen2) [inline .github/workflows/cloud-admin.yml:26]: I'm not sure if this is a problem or not, but I think the Applications team has all been on Node 16. (We should probably upgrade as mentioned [here](https://github.com/rilldata/rill-developer/issues/1916))

File: .github/workflows/cloud-admin.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+      - name: Build Cloud Admin
+        run: |-
+          npm install
+          npm run build -w web-admin
+
+      - name: Deploy ${{ matrix.name }} to Netlify Staging
```
Comment (begelundmuller) [inline .github/workflows/cloud-admin.yml:33]: Why `matrix.name` here? I don't see a matrix build

File: .github/workflows/cloud-admin.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy Cloud Admin to Netlify
+on:
+  push:
+    tags:
+      - "*"
+    branches: ["main"]
+    paths:
```
Comment (begelundmuller) [inline .github/workflows/cloud-admin.yml:7]: I don't think we need `paths` here – any tag on `main` can just trigger a build.

If you want to keep the `paths` – we also need to add `web-common` here

**Review by begelundmuller — CHANGES_REQUESTED** [review]

File: .github/workflows/cloud-admin.yml (hunk 2 above)
Comment (himadrisingh) [reply .github/workflows/cloud-admin.yml:10]: oh it should have been `common`

Comment (himadrisingh) [inline .github/workflows/cloud-admin.yml:10]:
```suggestion
      - "web-common/**"
```

File: .github/workflows/cloud-admin.yml (hunk 4 above)
Comment (himadrisingh) [reply .github/workflows/cloud-admin.yml:33]: copy paste error 🤦

Comment (himadrisingh) [inline .github/workflows/cloud-admin.yml:33]:
```suggestion
      - name: Deploy cloud-admin to Netlify
```

File: .github/workflows/cloud-admin.yml (hunk 3 above)
Comment (himadrisingh) [reply .github/workflows/cloud-admin.yml:26]: we have node 16 in the tests too
https://github.com/rilldata/rill-developer/blob/main/.github/workflows/web-test.yml#L31-L34

will move all to 16 then

File: .github/workflows/cloud-admin.yml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up NodeJS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```
Comment (himadrisingh) [inline .github/workflows/cloud-admin.yml:26]:
```suggestion
          node-version: 16
```

**Review by kaspersjo — APPROVED** [review]

**Review by begelundmuller — APPROVED** [review]

**Pull request merged**